### PR TITLE
Upgrade django to django 2.1 and python to python3

### PR DESCRIPTION
**Pull request by victorphoenix3**
Since the support for python2 ended in January it would be nice to upgrade the project to use python3 and django2

Ref: #58 #24

### CODE REVIEW

**Comment by victorphoenix3** [comment]
@goneall  @rtgdk I can run the project on my system. Please reviiew.

File: src/api/models.py [hunk 1]
```diff
@@ -60,3 +68,21 @@ class CheckLicenseFileUpload(models.Model):
     file = models.FileField(upload_to=user_directory_path)
     result = models.CharField(max_length=128,null=False,blank=False)
     status = models.IntegerField(default=200,blank=False)
+
+class SubmitLicenseModel(models.Model):
+
+    created = models.DateTimeField(auto_now_add=True)
+    owner = models.ForeignKey(User, to_field='id', on_delete=models.CASCADE)
```
Comment (victorphoenix3) [inline src/api/models.py:75]: I have used on_delete=models.CASCADE here and as well as at other places.

File: requirements.txt [hunk 2]
```diff
@@ -1,10 +1,13 @@
 Django==2.1.2
-djangorestframework==3.8.2
-JPype1==0.6.3
-pytz==2018.5
+jpype1==0.6.3
+numpy==1.16.4
```
Comment (rtgdk) [inline requirements.txt:3]: Why numpy?

Comment (victorphoenix3) [reply requirements.txt:3]: @rtgdk it is in the requirments.txt of the python2 version. Do we not need it?

Comment (rtgdk) [reply requirements.txt:3]: My bad, was looking at the displayed diff, should have checked diff b/w master and this branch.

File: src/api/models.py (hunk 1 above)
Comment (rtgdk) [reply src/api/models.py:75]: @victorphoenix3 Give me some time to review this, need to compare both master and python3 branch. In the meantime, you can update the branch with master (some small gitignore and readme changes went in last week). I also don't see travis.yml changes in this branch. The readme should have python3 instructions.

**Comment by rtgdk** [comment]
Opened a new PR with updated master and many other changes to support python3 - https://github.com/spdx/spdx-online-tools/pull/205
Thanks @victorphoenix3 for all the work on this.

**Comment by rtgdk** [comment]
Closed with #205, thanks for the initial work!

**Pull request closed without merging**